### PR TITLE
GLTFExporter credits where it's due

### PR DESCRIPTION
**Pull request by fernandojsg**

### CODE REVIEW

File: examples/js/exporters/GLTFExporter.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 /**
  * @author fernandojsg / http://fernandojsg.com
+ * @author Don McCurdy / https://www.donmccurdy.com
```
Comment (mrdoob) [inline examples/js/exporters/GLTFExporter.js]: Shouldn't it be `donmccurdy` and `takahirox`?

Comment (fernandojsg) [reply examples/js/exporters/GLTFExporter.js]: @mrdoob I just copied it from https://github.com/mrdoob/three.js/blob/dev/examples/js/loaders/GLTFLoader.js
@takahirox @donmccurdy any preference?

Comment (takahirox) [reply examples/js/exporters/GLTFExporter.js]: I prefer Takahiro

Comment (mrdoob) [reply examples/js/exporters/GLTFExporter.js]: I tend to use the username to it's easier to find on github. But I don't have strong opinions about it.

**Review by takahirox — APPROVED** [review]

File: examples/js/exporters/GLTFExporter.js (hunk 1 above)
Comment (takahirox) [reply examples/js/exporters/GLTFExporter.js]: In my case, the link is to github account page, so easy to find hehe :D

**Comment by mrdoob** [comment]
Thanks!

**Comment by donmccurdy** [comment]
Thanks! 🙂

**Pull request merged**